README.md

```diff
@@ -1,9 +1,8 @@
 # Eclipse Che documentation project
-[![Contribute](https://www.eclipse.org/che/contribute.svg)](https://che.openshift.io/#https://github.com/eclipse/che-docs)
+[![Contribute](https://www.eclipse.org/che/contribute.svg)](https://workspaces.openshift.com/f?url=https://github.com/eclipse/che-docs)
```

Comment:
@ibuziuk is the format `https://workspaces.openshift.com/#https://github.com/eclipse/che-docs` supposed to work in CRW 2.7?

Reply:
@l0rd `#` has never worked to my knowledge

Reply:
Maybe `#` only works with the new dashboard? cc @olexii4  @akurinnoy

Reply:
it's only with the new dashboard (and it's not yet in CRW)

Reply:
Yesterday I was requested to rollback the `#`. I understand this is a Dashboard Next related feature that will ship only with CRW 2.8, so not available now on workspaces.openshift.com.

Reply:
isn't it backward compatible? can we stick to `f?url=` for now?

Reply:
yep. Then a new PR to change it. I will merge it now. Will create a separate issue for the https://raw.githubusercontent.com/che-samples/che-theia-branding-example/master/devfile.yaml issue.